exercises/practice/forth/forth_test.go

```diff
@@ -6,23 +6,20 @@ import (
 )
 
 func TestForth(t *testing.T) {
-	for _, tg := range testGroups {
-		for _, tc := range tg.tests {
-			if v, err := Forth(tc.input); err == nil {
-				var _ error = err
+	for _, tc := range testCases {
+		t.Run(tc.description, func(t *testing.T) {
+			v, err := Forth(tc.input)
+			if err == nil {
+				var _ = err
```

Comment:
What is this line (13) for?

Reply:
To be honest I don't know. I think I left it there because it existed before I touched the file. I removed it know, as it seems not to be needed